test/metrics/metrics_test.go

```diff
@@ -473,15 +453,15 @@ func TestMetricsWhenUsersBanned(t *testing.T) {
 		err = hostAwait.WaitUntilSpaceAndSpaceBindingsDeleted(t, bannedUser.GetName())
 		require.NoError(t, err)
 		// verify the metrics
-		metricsAssertion.WaitForMetricDelta(t, UserSignupsMetric, 0)                                          // unchanged: user signup already existed
-		metricsAssertion.WaitForMetricDelta(t, UserSignupsApprovedMetric, 1)                                  // user approved
-		metricsAssertion.WaitForMetricDelta(t, UserSignupsApprovedWithMethodMetric, 0, "method", "automatic") // unchanged: unbanning uses previous method of approval
-		metricsAssertion.WaitForMetricDelta(t, UserSignupsApprovedWithMethodMetric, 1, "method", "manual")    // unbanning uses previous method of approval
-		metricsAssertion.WaitForMetricDelta(t, UserSignupsBannedMetric, 0)                                    // unchanged: banneduser already existed
-		metricsAssertion.WaitForMetricDelta(t, MasterUserRecordsPerDomainMetric, 1, "domain", "external")
-		metricsAssertion.WaitForMetricDelta(t, MasterUserRecordsPerDomainMetric, 0, "domain", "internal")
-		metricsAssertion.WaitForMetricDelta(t, SpacesMetric, 1, "cluster_name", memberAwait.ClusterName)  // space provisioned on member1
-		metricsAssertion.WaitForMetricDelta(t, SpacesMetric, 0, "cluster_name", memberAwait2.ClusterName) // no spaces on member2
+		hostAwait.WaitForMetricDelta(t, wait.UserSignupsMetric, 1)                                          // unchanged: user signup already existed
+		hostAwait.WaitForMetricDelta(t, wait.UserSignupsApprovedMetric, 2)                                  // user approved
+		hostAwait.WaitForMetricDelta(t, wait.UserSignupsApprovedWithMethodMetric, 0, "method", "automatic") // unchanged: unbanning uses previous method of approval
+		hostAwait.WaitForMetricDelta(t, wait.UserSignupsApprovedWithMethodMetric, 2, "method", "manual")    // unbanning uses previous method of approval
+		hostAwait.WaitForMetricDelta(t, wait.UserSignupsBannedMetric, 1)                                    // unchanged: banneduser already existed
```

Comment:
same as above, I'm not sure I understand why now those values should be different.

Reply:
same response as above :)